Added Makefile (#48)
bryzgaloff/airflow-clickhouse-plugin: README.md

    the `Client.__init__` will receive `secure=True` keyword argument in
    addition to other non-empty connection attributes.

#### Compression

You should install several packages to support compression. For example, for lz4:

> ```

Reviewer: ```suggestion
```bash
```

pip3 install clickhouse-cityhash lz4
```

> Then you should pass compression parameter `extra={"compression":"lz4"}`

Reviewer: Where to pass the parameter? Make an example more explicit please.

> Then you should pass compression parameter `extra={"compression":"lz4"}`

Reviewer: ```suggestion
Then you should pass compression parameter `extra={"compression":"lz4"}`.
```

> You can get additional information from [official documentation of clickhouse-driver](https://clickhouse-driver.readthedocs.io/en/latest/installation.html#installation-pypi)

Reviewer: ```suggestion
Refer to an [official documentation of clickhouse-driver][ch-driver-docs] for further details.
```

Reuse the existing links.

## Unit tests

> From the root project directory: `make unit`

Reviewer: Keep both options:

```suggestion
From the root project directory:

* Using Python: `python -m unittest discover -s tests/unit`
* Or using `make`: `make unit`
```

## Integration tests

Integration tests require access to ClickHouse server. Tests use connection
    URI defined [via environment variable][airflow-conn-env]
    `AIRFLOW_CONN_CLICKHOUSE_DEFAULT` with `clickhouse://localhost` as default.

> You can start clickhouse server locally with the following command:

Reviewer: ```suggestion
You can run ClickHouse server in a local Docker container using the following command:
```

```bash
make run-clickhouse
```

> And then run from the project root: `make integration`

Reviewer: How will this work? Will you attach to a docker-run container and run the command inside? So far I expect that `run-clickhouse` runs a docker container and `integration` still executes the commands locally. Am I wrong?

## All tests

> From the root project directory: `make tests`

Reviewer: Keep both options

Run ClickHouse server inside Docker:

```bash

> docker exec -it $(docker run --rm -d clickhouse/clickhouse-server) bash

Reviewer: You may also add your alternative for `make` here

```bash
apt-get update
apt-get install -y python3.10 python3-pip git make
git clone https://github.com/whisklabs/airflow-clickhouse-plugin.git
cd airflow-clickhouse-plugin

> make tests

Reviewer: Please keep both alternatives, i.e.:

```markdown
Using `python`:
(prev commands)

Using `make`:
(your new commands)
```
